python/tvm/relay/frontend/common.py

```diff
@@ -997,3 +998,57 @@ def try_resolve_var_to_const(x, graph_params):
         return _op.const(value, dtype)
 
     return x
+
+
+# Span filling to Relay for TFLite taken from
+# commit https://github.com/apache/tvm/pull/9723
```

Comment:
Hi @lhutton1, thank you for reviewing!
We've corrected a few points based on your previous comments.



> Its seems like this implementation was reverted in #10072, but #13402 looks to be achieving something similar, do you think it would be possible to rebase this PR ontop of #13402?

Checked the changes in #13402, some are the same as our PR and some are different, so I think it may be worth waiting for #13402 to be accepted?

Reply:
Thanks, yep I think that would be best, it seems like the review is almost finished as well :)